templates/keepalived.conf.j2

```diff
@@ -13,6 +13,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+{% if keepalived_global_defs is defined %}
+global_defs {
+{% for def in keepalived_global_defs.items() %}
```

Comment:
if keepalived_global_defs is a list, we should just iterate over list elements:
```
{% for def in keepalived_global_defs %}
```

Reply:
Oh, good point.